jobs/generation/Utilities.groovy

```diff
@@ -189,15 +189,6 @@ class Utilities {
                                 // For outerloop runs.
                                 'outer-latest-or-auto':'ubuntu1610-20170216-outer',
                                 ],
-                            'OSX10.11' :
-                                [
-                                // Generic version label
-                                '' : 'mac',
```

Comment:
I imagine no one is using this one, so we are not going to brake them with this change, right?